app/xevd_app.c

```diff
@@ -49,19 +49,32 @@ static void print_usage(void)
     }
 }
 
+/* read 'nal_unit_length' syntax in Annex-B of EVC standard */
+static int read_nalu_len(void * buf)
```

Comment:
Shouldn't that be part of public API? I mean read_nalu_len function.
XEVD clients have to read NAL unit sizes from EVC stream while decoding EVC stream. 
The only place that provides the information on how to do in correct way is the source code of xevd_app reference application. 
Do you think it's enough?

Reply:
Ok, I will try to add a new API named "xevd_info( )" to get brief information of input bitstream.
I think we can give nal unit length if it is annex-b format, and several more important information.
Please wait a little bit.

Reply:
@dkozinski 

Hi, I added new API for getting brief information of input bitstream.
Review again please.
Espeically, review the funciton "read_bitstream()" in xevd_app.c would be good for understanding.